pyteal/ast/abi/type.py

```diff
@@ -1,11 +1,11 @@
 from typing import TypeVar, Generic, Callable, Final, cast
 from abc import ABC, abstractmethod
 
-from pyteal.errors import TealInputError
-from pyteal.types import TealType
 from pyteal.ast.expr import Expr
-from pyteal.ast.scratchvar import ScratchVar
+from pyteal.ast.scratchvar import ScratchVar, AbstractVar
```

Comment:
```suggestion
from pyteal.ast.abstractvar import AbstractVar
from pyteal.ast.scratchvar import ScratchVar
```

though you may wish to re-sort the imports in this section

Reply:
good find, oh boy 🤦

Reply:
There's actually more wrong imports of `AbstractVar`, and I got them fixed.